### PR TITLE
SAK-47897 WC SakaiGrader add previous instructor feedback to new Grader after a resubmission

### PR DESCRIPTION
**Pull request by danielmerino**
…submission

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js [hunk 1]
```diff
@@ -589,6 +589,12 @@ export const graderRenderingMixin = Base => class extends Base {
                       <div class="grader-history-block">${unsafeHTML(this._submission.history.grades)}</div>
                     </div>
                   ` : nothing }
+                  ${this.submission.history.feedback_comment ? html`
+                    <div id="grader-history-feedback-wrapper">
+                      <div class="grader-history-title">${this.i18n.instructor_feedback}</div>
+                      <div class="grader-history-block">${unsafeHTML(this.submission.history.feedback_comment)}</div>
+                    </div>
+                  ` : nothing }
```
Comment (bgarciaentornos) [inline webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js:597]:
```suggestion
                  ${this._submission.history.feedback_comment ? html`
                    <div id="grader-history-feedback-wrapper">
                      <div class="grader-history-title">${this.i18n.instructor_feedback}</div>
                      <div class="grader-history-block">${unsafeHTML(this._submission.history.feedback_comment)}</div>
                    </div>
                  ` : nothing }
```

Comment (danielmerino) [reply webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js]: Done, thanks!

File: webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js [hunk 2]
```diff
@@ -589,6 +589,12 @@ export const graderRenderingMixin = Base => class extends Base {
                       <div class="grader-history-block">${unsafeHTML(this._submission.history.grades)}</div>
                     </div>
                   ` : nothing }
+                  ${this._submission.history.feedback_comment ? html`
```
Comment (jesusmmp) [inline webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js:592]: Should be this._submission.history.feedbackComment?

Comment (danielmerino) [reply webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js]: Aghh, done...

**Review by jesusmmp — APPROVED** [review]

**Review by ottenhoff — APPROVED** [review]
Thanks

**Pull request merged**